services/alert/service.go

```diff
@@ -1028,6 +1033,14 @@ func (s *Service) createHandlerActionFromSpec(spec HandlerActionSpec) (ha handle
 		}
 		h := s.PagerDutyService.Handler(c, s.logger)
 		ha = newPassThroughHandler(h)
+	case "pushover":
+		c := pushover.HandlerConfig{}
```

Comment:
Removed the `DefaultHandlerConfig` method since it wasn't really doing anything.